app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -22,6 +23,7 @@ class GenerateTokenController extends Controller
      */
     public function __construct(private RemoteUserLookup $userLookup)
     {
+        $this->systemUser = User::where('name', 'SYSTEM')->first();
```

Comment:
Can't we use `null` as system?

Reply:
we could, but I would like to make some distinction between cronjob, backend and more.

Reply:
Then `SYSTEM` wouldn't be a correct naming :P
Maybe we need to think of something else here. I don't like the fact that things like system etc are `User`s. That feels weird.
Maybe an extra field called `source` or something?